frontend/src/screens/learner/card.tsx

```diff
@@ -128,19 +143,28 @@ export const StudyCard: React.FC<StudyCardProps & { onSelect(study: ParticipantS
             data-is-completed={!!study.completedAt}
             onClick={onClick}
         >
-            <Image.image name={Image.title} height="200px" css={{ border: `1px solid ${colors.lightGray}`, borderRadius: 8 }} />
+            <Image.image
+                name={Image.title}
+                height={useIsMobileDevice() ? '97px' : '200px'}
```

Comment:
instead of using isMobile, we could use media.mobile in the css.  Also 97px seems oddly specific, guessing that was needed for something, but it makes me nervous that things will overflow on different browsers 🤔

Reply:
yeah 97px came from the figma design... we can iterate on this with iris/viki down the line if needed